cmd/bucket-handlers-listobjects.go

```diff
@@ -70,8 +70,15 @@ func (api objectAPIHandlers) ListObjectsV2Handler(w http.ResponseWriter, r *http
 		return
 	}
 
+	urlValues := r.URL.Query()
+
 	// Extract all the listObjectsV2 query params to their native values.
-	prefix, token, startAfter, delimiter, fetchOwner, maxKeys, _ := getListObjectsV2Args(r.URL.Query())
+	prefix, token, startAfter, delimiter, fetchOwner, maxKeys, _, errCode := getListObjectsV2Args(urlValues)
```

Comment:
One more comment, although unrelated to the specific issue this PR resolves, it is in the same `getListObjectsV2Args()` function.
`ListObjectsV2` command argument, `encodingType`, is collected and returned in `getListObjectsV2Args()` function, but then it is ignored/not used when called in `ListObjectsV2Handler()` function. 
So, it makes sense to clean it up from `getListObjectsV2Args()` and fix the calls to it in `TestListObjectsV2Resources()` and `ListObjectsV2Handler()` function definitions accordingly.
I agree if you argue that this needs to be addressed in a separate issue. However, since we make changes in the same function, I thought it is ok to piggyback this clean-up change here as part of this PR.

Reply:
Good observation @ebozduman but we can let it remain as it is part of S3 spec. We might have to implement encodingType in future